docs/architecture/grpc-for-wcf-developers/wcf-bindings.md

```diff
@@ -1,11 +1,35 @@
 ---
 title: WCF bindings and transports - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: Learn how the various WCF bindings and transports compare to gRPC.
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # WCF bindings and transports
 
+WCF has lots of different built-in *bindings* specifying different network protocols, wire formats, and other implementation details. gRPC effectively has just one network protocol and one wire format. However, you're still likely to discover that gRPC offers the best solution in most cases. What follows is a short discussion about the most relevant WCF bindings and how they compare to their equivalent in gRPC.
+
+## NetTCP
+
+WCF's NetTCP binding allows for persistent connections, small messages, and two-way messaging but only works between .NET clients and servers. gRPC allows the same functionality but based on an open standard. gRPC has almost all the features of WCF NetTCP binding, although not always in the same way. For example, in WCF, encryption is controlled through configuration and handled in the framework. In gRPC, encryption is achieved at the connection level using HTTP/2 over SSL/TLS.
+
+## HTTP
+
+WCF is only cross-platform when using SOAP over HTTP/1.1. WCF HTTP binding is text based, uses SOAP as the wire format, and is very slow by the standards of modern networked applications. It's only really used to provide cross-platform interoperability, or connection over internet infrastructure. The equivalent in gRPC—because it uses HTTP/2 as the underlying transport layer with the binary Protobuf wire format for messages—can offer Net TCP service level performance but with full cross-platform interoperability with all modern programming languages and frameworks.
+
+## Named Pipes
+
+WCF provided a Named Pipes binding for communication between processes on the same physical machine. Named Pipes is a proprietary Windows feature and doesn’t work like TCP, so HTTP/2 doesn’t work over Named Pipes, meaning gRPC can’t work over Named Pipes.
+
+Outside Windows, the functionality provided by Named Pipes is instead generally provided by Unix Domain Sockets. These sockets are regular TCP-like sockets represented with file-system addresses, such as `/var/run/docker.sock`, which gRPC can work with as both client and server. If you need to use Named Pipes style functionality on Windows, the next update to Windows 10 and Windows Server, in Fall 2019, adds Domain Sockets as a fully supported native feature within Windows. So, gRPC services running on these and later versions of Windows (or on Linux) can use Domain Sockets instead of Named Pipes. However, if your team is unable to update to the latest version of Windows, then you'll need to use localhost TCP addresses. Security concerns about using local TCP sockets can be alleviated with the use of certificate authentication between client and server.
```

Comment:
Fall 2019 has already finished in the southern hemisphere. Can you be more specific about the date? Maybe talk about in terms of quarters?

alleviate is considered a formal word. Cam you use a less formal synonym?